### PR TITLE
Added fix for rt_test.

### PR DESCRIPTION
**Pull request by poojabs7**
Changed distro name, and correct link to download the tarfile.

Signed-off-by: poojabs <pooja@oc2013156016.ibm.com>

### CODE REVIEW

**Comment by PraveenPenguin** [comment]
@poojabs7 please rebase as update your copy with upstream

File: perf/rt_tests.py [hunk 1]
```diff
@@ -32,14 +32,14 @@
 from avocado.utils import distro
 
 
-class rt_tests(Test):
+class RtTest(Test):
 
     def setUp(self):
         # Check for basic utilities
-        sm = SoftwareManager()
+        smm = SoftwareManager()
```
Comment (nasastry) [inline perf/rt_tests.py]: Reason for changing object name from `sm` to `smm`?

File: perf/rt_tests.py [hunk 2]
```diff
@@ -49,19 +49,22 @@ def setUp(self):
...
             deps.append("numactl-devel")
         for package in deps:
-            if not sm.check_installed(package) and not sm.install(package):
+            if not smm.check_installed(package) and not smm.install(package):
                 self.error(package + ' is needed for the test to be run')
-        locations = ["https://kernel.googlesource.com/pub/scm/utils/rt-tests/"
-                     "rt-tests/+archive/master.tar.gz"]
-        tarball = self.fetch_asset("rt-tests.tar.gz", locations=locations)
+        tarball = self.fetch_asset("https://www.kernel.org/pub/linux/utils/"
+                                   "rt-tests/rt-tests-1.0.tar.gz", expire='7d')
```
Comment (nasastry) [inline perf/rt_tests.py]: Can this URL be added to a cfg file?

File: perf/rt_tests.py (hunk 1 above)
Comment (PraveenPenguin) [inline perf/rt_tests.py]: i think should fine as pylint complains if variable length less than 3 

C: 39, 8: Invalid variable name "sm" (invalid-name)

File: perf/rt_tests.py (hunk 2 above)
Comment (PraveenPenguin) [inline perf/rt_tests.py]: i think you meant about yaml file

Comment (nasastry) [inline perf/rt_tests.py]: Yes.

File: perf/rt_tests.py (hunk 1 above)
Comment (nasastry) [inline perf/rt_tests.py]: Ok. Thanks for clarification

**Comment by poojabs7** [comment]
The fix i salready sent by harish-24 .So closing this PR.
https://github.com/avocado-framework-tests/avocado-misc-tests/blob/master/perf/rt_tests.py

**Pull request closed without merging**